### ISSUE TITLE
[P2P2]  ERROR   cn.block_queue  src/cryptonote_protocol/cryptonote_protocol_handler.h:96        Unknown command:1020

### ISSUE THREAD

**Issue by freebet** [graft-project/GraftNetwork#144]
[P2P2]  ERROR   cn.block_queue  src/cryptonote_protocol/cryptonote_protocol_handler.h:96        Unknown command:1020

any idea?

**Comment by TheBudda** [comment]
I got the same error even when I am re-sinking the blockchain!
[P2P2]  ERROR   cn.block_queue  src/cryptonote_protocol/cryptonote_protocol_handler.h:96        Unknown command:1020

**Comment by EDDragonWolf** [comment]
Did you take source code from the master? Also what network you try to sync (mainnet, public testnet, etc.)?

**Comment by TheBudda** [comment]
@EDDragonWolf  I started to get the error above when I checked my graftnoned screen. I have now reinstalled Ubuntu 18.04. I am now compiling Graft 1.2.2. I was sync on the mainet running CryptoBlock UK pools. Everything has been fine for weeks and we found some blocks!

**Comment by TheBudda** [comment]
A refresh install of Ubuntu 18.04 on a dedicated server from OVH. I have just started to sync the blockchain and I get this error! Please see the screenshot attached

[P2P2] ERROR cn.block_queue src/cryptonote_protocol/cryptonote_protocol_handler.h:96 Unknown command:1020)


![screenshot 2018-08-29 11 22 27](https://user-images.githubusercontent.com/16983376/44782219-698edf00-ab7e-11e8-918b-06d5b59498d7.png)
!!

**Comment by TheBudda** [comment]
![screenshot 2018-08-29 11 28 07](https://user-images.githubusercontent.com/16983376/44782291-a65ad600-ab7e-11e8-8d21-71b28f0a406d.png)

**Comment by EDDragonWolf** [comment]
Thanks, we will check it, but don't worry this error means that your node cannot process request from another node. Look like somebody was linked test version of the node to mainnet.

**Comment by TheBudda** [comment]
yeah, crazy amount of errors. I do get why! My pool was running sweet for weeks
![screenshot 2018-08-29 12 35 05](https://user-images.githubusercontent.com/16983376/44785227-1ae64280-ab88-11e8-8f60-ae6f14b7b90b.png)

**Comment by TheBudda** [comment]
Crazy amount of errors!! 

![screenshot 2018-08-29 13 59 13](https://user-images.githubusercontent.com/16983376/44789143-c943b500-ab93-11e8-8088-3f24dc99aebd.png)

**Comment by TheBudda** [comment]
212 % | ./graftnoded it's killing my server
